.travis.yml

```diff
@@ -25,25 +26,23 @@ before_script:
   - if [[ $DATABASE_BACKEND == 'mysql_innodb' ]]; then mysql -e 'create database pootle CHARACTER SET utf8 COLLATE utf8_general_ci;'; fi
   - if [[ $DATABASE_BACKEND == 'mysql_innodb' ]]; then mysql -e "SET GLOBAL wait_timeout = 36000;"; fi
 script:
-  - python -m compileall -q -f .
-  - make docs
-  # FIXME revert breakdown of 'make build' after core merge
-  #- make build SHELL=/bin/bash TAIL='|tail -40; exit $$PIPESTATUS'
-  - make assets
-  - make mo
-  - DJANGO_SETTINGS_MODULE=pootle.settings ./setup.py build_checks_templates
-  - ./manage.py migrate --noinput --traceback
-  - ./manage.py migrate | (egrep "Your models have changes" && exit 1 || exit 0)
-  - ./manage.py initdb --traceback --no-projects
-  - ./manage.py runserver --traceback &
-  - TESTPID=$!
-  - sleep 10
-  - kill -2 $TESTPID
+  - if [[ $META ]]; then python -m compileall -q -f .; fi
+  - if [[ $META ]]; then make docs; fi
+  - if [[ $META ]]; then make mo; fi
+  - if [[ $META ]]; then make pep8; fi
+  # - if [[ $META ]]; then pep257; fi
+  - if [[ ! $META ]]; then make assets; fi
+  - if [[ ! $META ]]; then DJANGO_SETTINGS_MODULE=pootle.settings ./setup.py build_checks_templates; fi
```

Comment:
I see $META as not part of the core code.  So no don't want to pull that out just yet.